pkg/ccl/sqlproxyccl/connector.go

```diff
@@ -156,7 +156,7 @@ func (c *connector) OpenTenantConnWithAuth(
 	}
 	defer func() {
 		if retErr != nil {
-			serverConn.Close()
+			_ = serverConn.Close()
```

Comment:
I didn't catch that in the main PR. Don't you want to use errors.CombineErrors here?

Reply:
Yes. I added the call to CombineErrors